build/antrea/ovs/Dockerfile.ubi

```diff
@@ -0,0 +1,50 @@
+FROM registry.access.redhat.com/ubi8 as base-rpms
+
+COPY CentOS.repo /tmp/CentOS.repo
+RUN rm -f /etc/yum.repos.d/* && mv /tmp/CentOS.repo /etc/yum.repos.d/CentOS.repo && \
+    curl https://www.centos.org/keys/RPM-GPG-KEY-CentOS-Official -o /etc/pki/rpm-gpg/RPM-GPG-KEY-centosofficial && \
+    subscription-manager config --rhsm.manage_repos=0 && \ 
+    yum clean all -y && yum update -y
+
+# Download dependencies for ovs.
+RUN yum install -y epel-release && \ 
+    yum install -y --downloadonly --downloaddir="/tmp/build-packages" unbound python3-sphinx desktop-file-utils groff graphviz selinux-policy-devel && \
+    yum install -y --downloadonly --downloaddir="/tmp/install-packages" strongswan libreswan
+RUN yum install -y --downloadonly --downloaddir="/tmp/install-packages" unbound
+
+FROM registry.access.redhat.com/ubi8 as ovs-rpms
+
+# Some patches may not apply cleanly if another version is provided.
+ARG OVS_VERSION=2.14.0
+
+COPY --from=base-rpms /tmp/build-packages/* /tmp/packages/
+COPY apply-patches.sh /
```

Comment:
May I know why we need apply-patches.sh and charon-logging.conf to install OVS?

Reply:
> May I know why we need apply-patches.sh and charon-logging.conf to install OVS?

Generally it installs patches for ovs, i just copy it from Antrea repo and don't know much details. Maybe check the comments in apply-patches.sh for details.